locales/ru.json

```diff
@@ -1098,7 +1098,11 @@
     "td-headers-attr": {
       "pass": "Атрибут headers используется исключительно для ссылки на другие ячейки таблицы",
       "incomplete": "Атрибут headers пуст",
-      "fail": "Атрибут headers не используется исключительно для ссылки на другие ячейки таблицы"
+      "fail": {
```

Comment:
I tried to update locales. I found that all locales are quite outdated. Not sure that this change is good place to bring all locales up to date. So i updated only "ru" which i contributed recently. Not sure what is contribution policy here.